sig-node/README.md

```diff
@@ -53,6 +53,13 @@ The following [working groups][working-group-definition] are sponsored by sig-no
 ## Subprojects
 
 The following [subprojects][subproject-definition] are owned by sig-node:
+### ci-testing
+- **Owners:**
+  - [kubernetes/kubernetes/test/e2e_node](https://github.com/kubernetes/kubernetes/blob/master/test/e2e_node/OWNERS)
+  - [kubernetes/test-infra/config/jobs/kubernetes/sig-node](https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-node/OWNERS)
+  - [kubernetes/test-infra/jobs/e2e_node](https://github.com/kubernetes/test-infra/blob/master/jobs/e2e_node/OWNERS)
```

Comment:
We can also mention:

- config/jobs/cadvisor/OWNERS
- config/jobs/kubernetes/node-problem-detector/OWNERS

in the test-infra project. Those are optional, but since projects are under the SIG Node ownership, would be good to get them into the list for completeness

Reply:
alternatively those can be added into the corresponding subprojects

Reply:
Yeah they should be added to their respective subprojects - I'll do a follow up PR for those